coops/sentinels.tex

```diff
@@ -27,8 +27,10 @@ \subsection*{\MakeUppercase{Player Setup}}
 \textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}
 
 \textbf{Starting Units:}
-- A pack of \svg{bronze} units with the \textit{lowest} recruitment cost.
-- A few \svg{bronze} units with the \textit{highest} recruitment cost.
+\begin{itemize}
+  \item A Pack of \svg{bronze} units with the \textit{lowest} recruitment cost.
+  \item A Few \svg{bronze} units with the \textit{highest} recruitment cost.
+\end{itemize}
```

Comment:
Why is this changed to `itemize` and this scenario has the previous format?

https://github.com/qwrtln/Homm3BG-mission-book/pull/9/files#diff-519a49998a7b2897ff9fcf1eddc8d64be20669fb7657020130f86b9ae17ad558R30

Reply:
Or maybe the other way around. Let's fix this PR to also have `itemize` for starting units :)

Reply:
And this is my last comment, I promise :D Well done!